src/importexport/imagesexport/iimagesexportconfiguration.h

```diff
@@ -33,13 +33,16 @@ class IImagesExportConfiguration : MODULE_EXPORT_INTERFACE
 
     // Pdf
     virtual int exportPdfDpiResolution() const = 0;
+    virtual void setExportPdfDpiResolution(const int dpi) const = 0;
 
     // Png
     virtual float exportPngDpiResolution() const = 0;
-    virtual bool exportPngWithTransparentBackground() const = 0;
 
     //! NOTE Maybe set from command line
     virtual void setExportPngDpiResolution(std::optional<float> dpi) = 0;
+
+    virtual bool exportPngWithTransparentBackground() const = 0;
+    virtual void setExportPngWithTransparentBackground(const bool transparent) const = 0;
```

Comment:
Better to remove all const from this method:
virtual void setExportPngWithTransparentBackground(bool transparent) = 0;